examples/fundamentals/data_container.myst.md

```diff
@@ -58,19 +58,17 @@ This notebook will illustrate each of these benefits in turn, and show you the b
 
 +++
 
-## Types of Data Containers
+## Current and past states of Data Containers
```

Comment:
I think this whole section can be deleted and maybe replaced with an info box of your last comment.

Reply:
I agree, I will make the change.